### PR TITLE
add post for BRIDGE paper

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

**Comment by erasche** [comment]
Can one of the admins verify this patch?

File: _posts/2019-03-15-bridge.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+site: freiburg
+title: "New Paper - Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE)"
+tags: [paper]
+doi: "10.1093/bioinformatics/btz107"
+supporters:
+ - denbi
```
Comment (simonbray) [inline _posts/2019-03-15-bridge.md]: Wasn't sure what to add here, so put the same as for the most recent paper post.

**Comment by bgruening** [comment]
fixes: https://github.com/usegalaxy-eu/website/issues/191

**Comment by bgruening** [comment]
спасибо

**Comment by simonbray** [comment]
пожалуйста! ;)

**Pull request merged**